lib/video_info/providers/youtube_scraper.rb

```diff
@@ -68,7 +69,7 @@ def available?
         data.css('div#unavailable-submessage').text.strip.empty?
       end
 
-      def _set_data_from_api(api_url = _api_url)
+      def _set_data_from_api_impl(api_url = _api_url)
```

Comment:
Assignment Branch Condition size for _set_data_from_api_impl is too high. [22.29/15]